### PR TITLE
Adding custom data field (Issue #65) to Linux OS profiles.

### PR DESCRIPTION
**Pull request by dahlke**
For initialization.

### CODE REVIEW

File: test/fixture/terraform.tfvars [hunk 1]
```diff
@@ -5,3 +5,4 @@ vm_os_simple_1      = "UbuntuServer"
 vm_os_simple_2      = "Debian"
 admin_username      = "azureuser"
 admin_password      = "P@ssw0rd12345!"
+custom_data         = ""
```
Comment (metacpp) [inline test/fixture/terraform.tfvars]: new line.

**Comment by metacpp** [comment]
@dahlke Thanks for the contribution ! Most of them look good, left some super minor comments.

**Comment by dahlke** [comment]
@metacpp no problem!

**Comment by dahlke** [comment]
Great @JunyiYi! Is it possible to get it merged in?

**Comment by metacpp** [comment]
@dahlke thanks for the contribution !

**Pull request merged**